config/initializers/cors.rb

```diff
@@ -5,12 +5,12 @@
 
 # Read more: https://github.com/cyu/rack-cors
 
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins "example.com"
-#
-#     resource "*",
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "*"
```

Comment:
Hey Joop, I think we should consider locking down the allowed origins to only allow CORS with the front end app hosted on Vercel.

Reply:
Hey Eliza, that makes sense thank you. We can go ahead and merge this PR, and we can lock down the origins for CORS later.